docs/src/snippets/button-size.mdx

```diff
@@ -0,0 +1,21 @@
+<!-- eslint-disable mdx/remark -->
+
+### Mobile vs. desktop size
+
+<!-- eslint-enable mdx/remark -->
```

Comment:
I'm not super happy about needing these in every snippet with a heading, but I don't know a way to disable a specific remark rule for a directory given our current set up. Do you, @silvenon? A separate `.remarkrc.js` isn't picked up the way the `.eslintrc.js` for the directory is.